### ISSUE TITLE
Extracting the Scraping  to a Service class

### ISSUE THREAD

**Issue by olivierobert** [waiyan93/web-scraping#12]
## Issue

Similarly to #8, `ScrapeGoolgeSearchResult` contains too much business logic, hence is hard to test. With #9, the class can be simplified by calling a new instance of a service class that would contain the scraping logic. `$host` and `$userAgents` could be defined as class attributes instead of config variables or local variables.

## Expected

`ScrapeGoolgeSearchResult` does not contain the web scraping logic.

**Comment by waiyan93** [comment]
Yeah sir. I will abstract that logic into a class. So that I can manage it easily.
